prompt.go

```diff
@@ -11,12 +11,12 @@ import (
 type PromptFunc func(string) (string, error)
 
 func TerminalPrompt(prompt string) (string, error) {
-	fmt.Printf("%s: ", prompt)
+	log.SetFlags(0)
+	log.Printf("%s: ", prompt)
```

Comment:
this will still output to standard outout because that is the default behavior of Printf